src/test/shared/cloudformation/cloudformation.test.ts

```diff
@@ -76,15 +86,31 @@ describe ('CloudFormation', () => {
             Environment:
                 Variables:
                     ENVVAR: envvar`
+        }
+
+        it('can successfully load a file', async () => {
+            const yamlStr = makeSampleSamTemplateYaml(true)
 
             await strToYamlFile(yamlStr, filename)
             const loadedTemplate = await CloudFormation.load(filename)
             assert.deepStrictEqual(loadedTemplate, createBaseTemplate())
         })
 
-        it ('can successfully load a file with parameters', async () => {
+        it('can successfully load a file without globals', async () => {
+            const yamlStr = makeSampleSamTemplateYaml(false)
+
+            await strToYamlFile(yamlStr, filename)
+            const loadedTemplate = await CloudFormation.load(filename)
+
+            const expectedTemplate = createBaseTemplate()
+            delete expectedTemplate.Globals
+
+            assert.deepStrictEqual(loadedTemplate, expectedTemplate)
+        })
+
+        it('can successfully load a file with parameters', async () => {
             const yamlStr: string =
-`Parameters:
+                `Parameters:
```

Comment:
Was this (and other similar lines) fixed by the linter?

Reply:
Our repo doesn't have the formatter activated currently.
VS Code's autoformatting made adjustments to the file when I saved it.